src/redux/services/api.ts

```diff
@@ -156,6 +158,12 @@ export const api = createApi({
       providesTags: (result, error, arg) => [{ type: 'Customer', id: arg }],
     }),
 
+    // GET: Fetch user latest applicant's details
+    getCustomerLatestApplicantInfo: builder.query<Applicant, string>({
+      query: (id) => `applicant/latest/${id}/`,
+      providesTags: (result, error, arg) => [{ type: 'Applicant', id: arg }],
+    }),
```

Comment:
When creating, viewing or editing a customer that does not have any applications, you'll get constantly the error toast: 

![image](https://github.com/user-attachments/assets/c1e22736-0552-4cf6-a0c8-9d1542ed096b)

`rtkQueryErrorLogger` doesn't give any mercy, so perhaps the best course would be to change the API response to 204 no content to avoid that?

Reply:
this was fixed on the backend